### PR TITLE
Do Not Merge: Showoff

### PR DESCRIPTION
**Pull request by nibalizer**
Work in progress of getting content moved into showoff.

### CODE REVIEW

**Comment by MHBauer** [comment]
at what point in time was this conversion done?

File: presentation/showoff/world/01_slide.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<!SLIDE >
+# My Presentation
```
Comment (MHBauer) [inline presentation/showoff/world/01_slide.md]: extra directory or an example?

File: presentation/showoff/hello/02_slide.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!SLIDE bullets incremental>
+# Bullet Points
+
+* first point
+* second point
```
Comment (MHBauer) [inline presentation/showoff/hello/02_slide.md]: same here, extra directory or an example?

File: presentation/showoff/Introduction/01_slide.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+!SLIDE[bg=_images/backgrounds/white_bg.png]
```
Comment (MHBauer) [inline presentation/showoff/Introduction/01_slide.md]: how late in the process can this be edited?

**Comment by duglin** [comment]
Can you add a Dockerfile to build an image for this? I keep trying to get this showoff stuff to work via Docker and still can't do it. It runs but the presentation never shows up in the browser.

**Comment by MHBauer** [comment]
`docker run --rm -v $(pwd):/app -w /app -p 9090:9090 ruby sh -c "gem install showoff && showoff serve" `
works for me when checking out 
`git@github.ibm.com:skrum/talk_2018_k8s_and_istio_at_lfnw.git`

**Comment by nibalizer** [comment]
@MHBauer I started last night.

File: presentation/showoff/Introduction/01_slide.md (hunk 3 above)
Comment (nibalizer) [reply presentation/showoff/Introduction/01_slide.md]: Unsure what you're getting at here. Can you clarify?

File: presentation/showoff/world/01_slide.md (hunk 1 above)
Comment (nibalizer) [reply presentation/showoff/world/01_slide.md]: Extra directory, will clean.

File: presentation/showoff/hello/02_slide.md (hunk 2 above)
Comment (nibalizer) [reply presentation/showoff/hello/02_slide.md]: Extra directory, will clean.

File: presentation/showoff/Introduction/01_slide.md (hunk 3 above)
Comment (MHBauer) [reply presentation/showoff/Introduction/01_slide.md]: Whoever's presenting should have their info at the front. Not sure if we're intending to publish on the web or have people live presenting it.

An attribution for us all at some point inside would be nice as well.

**Comment by stevemar** [comment]
Closing due to inactivity and conflicts

**Pull request closed without merging**